packages/mailer/handlers/in-memory/README.md

```diff
@@ -0,0 +1,3 @@
+# Mailer - Handler - In Memory
```

Comment:
Once in, I can work on the various readies to explain how and when to use. Like this one for testing.